### PR TITLE
Bootstrap theme list

### PR DESCRIPTION
**Pull request by ryestew**
List of bootstrap themes and their URL's in theme-chooser.js for use in the settings tab and to change to the dark or light theme in the editor.

### CODE REVIEW

**Comment by yann300** [comment]
could you squash it in one commit? `not quite working yet` is not really a commit here.
And remove all commented and unnecessary code

File: src/app/tabs/settings-tab.js [hunk 1]
```diff
@@ -45,6 +37,25 @@ module.exports = class SettingsTab extends ApiFactory {
       location: 'swapPanel'
     }
   }
+  createThemeCheckies () {
+    let themes = styleGuide.getThemes()
+    let self = this
```
Comment (ryestew) [inline src/app/tabs/settings-tab.js]: What is the better way of dealing with the scope here?

**Comment by yann300** [comment]
I think that is good to merge. please @LianaHus would you have a look?
just a quick question: what about the ace editor, do you keep the current behavior, or is this something yet to be done?

**Comment by ryestew** [comment]
1 more fix to do - it is not loading the saved theme from browser storage ( after a refresh).

**Pull request merged**